### PR TITLE
Add cypress husky and lint-staged

### PR DESCRIPTION
**Pull request by oterral**

### CODE REVIEW

File: components/Header.js [hunk 1]
```diff
@@ -60,13 +60,15 @@ export default function Header({ translationPath }) {
...
           <button
+            id="menuButton"
             className="lg:hidden text-blue pr-8 w-16"
             onClick={() => setMenuOpen(true)}
           >
             <MenuIcon />
           </button>
           {/* -------- Mobile Navigation -------- */}
           <div
+            id="menuListMobile"
```
Comment (friedjoff) [inline components/Header.js:71]: How about using `data-cy` attributes, like [documented here](https://docs.cypress.io/guides/references/best-practices#Selecting-Elements)? I like the clear separation between JS/CSS and Cypress.

File: cypress/integration/about.spec.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+describe("about", () => {
+  it("should navigate to the about page form the url", () => {
+    // Start from the index page
+    cy.visit("http://localhost:3000/about");
```
Comment (friedjoff) [inline cypress/integration/about.spec.js:4]: I think we should configure the base URL `http://localhost:300` in the Cypress config file.

**Review by friedjoff — CHANGES_REQUESTED** [review]

File: components/Header.js (hunk 1 above)
Comment (oterral) [reply components/Header.js:71]: Totally agree

**Review by friedjoff — APPROVED** [review]
LGTM!

**Pull request merged**